parameters.yml

```diff
@@ -174,11 +174,13 @@ asciidoc:
   - attribute: sp-base
     value: SP4
   - attribute:  sp-version
-    value: SP5 #use for normal text
+    value: SP4 #use for normal text
```

Comment:
Same as below, not clear to me why we need `sp-base` and `sp-version.

Reply:
One for the SP version SUSE Manager is based on (4.3 SP4), and the other latest supported SLE client (4.3 supports SP5)
IIRC, we want to switch to sp-server and sp-client for clarity.  I proposed it, Joseph agreed, but I hesitated to change it actually.

Reply:
So what should the be right value then? Whatever entity is used for clients should have a clear name for it.. or at least a clear comment. Does this need to stay as SP5?